DataManagementSystem/Agent/DataLoggingAgent.py

```diff
@@ -0,0 +1,26 @@
+'''
+Created on Jun 19, 2015
+
+@author: Corentin Berger
+'''
+from DIRAC import S_OK
+
+from DIRAC.Core.Base.AgentModule import AgentModule
+from DIRAC.DataManagementSystem.Client.test.mockDirac import ClientB
+
+# # agent's name
+AGENT_NAME = 'DataManagement/DataLoggingAgent'
+
+########################################################################
+class DataLoggingAgent( AgentModule ):
+  def initialize( self ):
```

Comment:
If this agent is just for test, it should be in a test directory... Do you mean really an agent or should be a script?